src/test/java/seedu/address/logic/parser/AddTaskCommandParserTest.java

```diff
@@ -88,6 +88,10 @@ public void parse_invalidValue_failure() {
                 + INVALID_TASK_STATUS_DESC
                 + TAG_DESC_FINANCE + TAG_DESC_PUBLICITY, TaskStatus.MESSAGE_CONSTRAINTS);
 
+<<<<<<< HEAD
```

Comment:
You should remove stuff like Head, Master etc.. this is from the merge conflict page